html/index.html

```diff
@@ -1 +1,48 @@
-<!DOCTYPE html><html><head> <meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>BH OnoFre</title> <link rel="icon" href="data:,"> <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport"> <link rel="stylesheet" href="css/styles.min.css?8.301"></head><body class=" skin-blue sidebar-mini"><div> <header class="main-header"> <nav class="navbar navbar-static-top" role="navigation"> <ul class="sidebar-menu" data-widget="tree"> <li class=""><a href="#" class="menu-item" data-menu="node"> <span class="lang-node">NÓ</span></a></li><li><a href="#" class="menu-item" data-menu="devices"> <span class="lang-features">FUNÇÕES</span></a> </li><li class=""><a href="#" class="menu-item" data-menu="wifi"> <span>WI-Fi</span></a></li><li class=""><a href="#" class="menu-item" data-menu="integrations"> <span><span class="lang-integrations">INTEGRAÇÕES</span></span></a></li></ul> </nav> </header> <div class="user-panel"> <div class="pull-left" style="color: white; height: 10px;"> <div style="padding: 5px;"><span style="color:#88bf9c;">Wi-Fi: </span> <span id="ssid_lbl"></span> <span id="wifi-signal">0%</span></div></div><div class="pull-left" style="color: white; height: 10px;"> <div style="padding: 5px;"><span style="color:#88bf9c;">MQTT: </span> <span id="mqtt_lbl"></span> <span id="mqtt-state">desligado</span></div></div><div class="pull-right" style="font-size: 14px; color: white;"> <button style="border-bottom-left-radius: 10px; background-color: #88bf9c; border: none;" onclick="loadsLanguage('PT');">PT </button> <button style="border-bottom-right-radius: 10px;background-color: #88bf9c; border: none;" onclick="loadsLanguage('EN');">EN </button> </div></div><div class="content-wrapper"> <section class="content container-fluid dashboard-container"></section> </div><footer class="main-footer"> <div class="pull-right hidden-xs"><span class="lang-version">Versão </span> <span id="version_lbl">-</span> </div><strong>BH OnOfre - Powered by <a href="https://github.com/brunohorta82/BH-Easy-Iot">Bruno Horta</a> @ 2018 in Portugal</strong></footer></div><script src="js/jquery.min.js"></script><script src="js/index.js?8.301"></script></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head> 
+<meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" content="IE=edge">
+<title>BH OnoFre</title>
+<link rel="icon" href="data:,"> 
+<meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport"> 
+<link rel="stylesheet" href="css/styles.min.css?8.301">
+
+<!-- Title Logo -->
```

Comment:
@brunohorta82 I add here to have a logo before the name but dont know exactly yet where and i can put it, if not we can delete it